lib/mix/tasks/clean_examples.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Mix.Tasks.CleanExamples do
```

Comment:
I had some test failures while running some of the tests that were fixed by removing the compiled test files so they could be re-compiled from scratch. I'm adding this mix task, along with an alias to call it before `mix test`, to automate the process so that others won't experience the same issues.